lib/web/apiserver_test.go

```diff
@@ -1875,6 +1873,14 @@ func TestWebSessionsRenew(t *testing.T) {
 	prevSessionID := decodeSessionCookie(t, prevSessionCookie.Value)
 	activeSessionID := decodeSessionCookie(t, sessionCookie.Value)
 	require.NotEmpty(t, cmp.Diff(prevSessionID, activeSessionID))
+	require.NotEmpty(t, cmp.Diff(pack.session.ExpiresIn, newPack.session.ExpiresIn))
+
+	s := auth.WebSession{
+		Expires: newPack.session.Session.Expires,
+		Roles:   []string{"foo"},
```

Comment:
I don't really understand how this is working here. 
There's a number of access request test cases in `auth/tls_test.go` - is there any intersection with this? Do you have to test this here as well?